### ISSUE TITLE
Slash as Division

### ISSUE THREAD

**Issue by maqnus** [folkehelseinstituttet/Fhi.Frontend.Style#99]
After introducing language-service in a angular-project, the build-log returns issues from the Fhi.Frontend.Style dependency with using slash as devision mark.

Sass currently treats / as a division operation in some contexts and a separator in others. This makes it difficult for Sass users to tell what any given / will mean, and makes it hard to work with new CSS features that use / as a separator.

My issues points towards:
- **_core.variables.scss:5 $fhi-px: 1rem/16;**
  _reccomended changed to $fhi-px: math.div(1rem, 16);_
- **_form-control.scss:12  border-radius: $fhi-input-height-xs / 2;**
   _reccomended changed to $fhi-px: math.div($fhi-input-height-xs , 2);_
- **_form-control.scss:39 border-radius: $fhi-input-height-lg / 2;**
   _reccomended changed to $fhi-px: math.div($fhi-input-height-lg , 2);_

ref: 
- **slash as division** https://sass-lang.com/documentation/breaking-changes/slash-div
- **Angular Language Service** https://angular.io/guide/language-service

**Comment by maqnus** [comment]
Screenshot from build-log
![image](https://user-images.githubusercontent.com/3446458/142618546-8e6fa5ed-e100-4abc-ae7a-bd4372f088da.png)

**Comment by proand** [comment]
I'll add the requested changes.

Just a side note @maqnus:
The language-service is introduced in [Fhi.Frontend.Demo](https://github.com/folkehelseinstituttet/Fhi.Frontend.Demo) as well, but I can't find this warning in the build log. Is there a difference in how we added the service? Here's how I did it in 
[tsconfig.json](https://github.com/folkehelseinstituttet/Fhi.Frontend.Demo/blob/cd5723d5e89425fd00672abd48dceb275433290f/tsconfig.json#L2)
